src/screens/ReceiveFundsSelectAccount.js

```diff
@@ -25,7 +25,8 @@ class ReceiveFundsSelectAccount extends Component<{
   navigation: NavigationScreenProp<{
     params: {
       selectedAccountId: string,
-      setAccountId: string => void
+      setAccountId: string => void,
+      syncCounterValue: string => void
```

Comment:
We shouldn't introduce another callback than just setAccountId. instead we should do the syncCounterValue logic on the other side.

Reply:
you can remove syncCounterValue from the prop type